sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/spark_func.py

```diff
@@ -108,65 +108,80 @@ def spark(
     resources: Optional[Union[Dict, SparkResourceConfiguration]] = None,
     **kwargs,
 ) -> Spark:
-    """Create a Spark object which can be used inside dsl.pipeline as a function and
+    """Creates a Spark object which can be used inside dsl.pipeline as a function and
```

Comment:
The chunk "can be used inside dsl.pipeline" is awkward. Can that be reworded somehow? I'm not quite sure what the intended meaning is, otherwise I'd offer a suggestion.

Reply:
Yeah, not sure why they used this wording initially. I've re-worded it like this and I'll add an example of it being used in the @dsl.pipeline function:

Creates a Spark object which can be used inside a dsl.pipeline function or used as a standalone Spark job.